apps/dh/api-dh/source/DataHub.WebApi/Registration/DomainRegistrationExtensions.cs

```diff
@@ -117,93 +106,139 @@ private static Uri GetBaseUri(string baseUrl)
             : new Uri("https://empty");
     }
 
-    private static IServiceCollection AddWholesaleClient(this IServiceCollection serviceCollection, Uri baseUri)
+    private static IServiceCollection AddWholesaleClient(this IServiceCollection serviceCollection)
```

Comment:
These 11 functions should be able to be written as 1 reusable.

Reply:
Good idea